config/locales/en.yml

```diff
@@ -83,7 +83,7 @@ en:
 
   shared:
     footer:
-      copyright_html: "&copy; Copyright 2010%{to_date} Northwestern University"
+      copyright_html: "&copy; Copyright 2010&ndash;%{to_date} Northwestern University"
```

Comment:
I was wondering if we should we remove the reference to Northwestern University here? But they started nucore, and we can easily override this in the forks.